corehq/apps/app_manager/detail_screen.py

```diff
@@ -517,6 +521,12 @@ class Address(HideShortColumn):
     template_width = 0
 
 
+@register_format_type('address-popup')
+class AddressPopup(HideShortColumn):
+    template_form = 'address-popup'
+    template_width = 0
```

Comment:
I don't think this is necessary, `HideShortColumn` should take care of this. (Same is true for the `Address` class, I just didn't notice previously.)